python/kwiver/vital/types/__init__.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+
+from kwiver.vital.types.types import *
+from kwiver.vital.types.bounding_box import *
+from kwiver.vital.types.camera import *
+from kwiver.vital.types.camera_intrinsics import *
+from kwiver.vital.types.camera_map import *
+from kwiver.vital.types.class_map import *
+from kwiver.vital.types.color import *
+from kwiver.vital.types.covariance import *
+from kwiver.vital.types.descriptor import *
+from kwiver.vital.types.descriptor_set import *
+from kwiver.vital.types.detected_object import *
+from kwiver.vital.types.detected_object_set import *
+from kwiver.vital.types.eigen import *
+from kwiver.vital.types.feature import *
+from kwiver.vital.types.homography import *
+from kwiver.vital.types.landmark import *
+from kwiver.vital.types.rotation import *
+from kwiver.vital.types.similarity import *
+from kwiver.vital.types.timestamp import *
+from kwiver.vital.types.track import *
+from kwiver.vital.types.track_set import *
+from kwiver.vital.types.object_track_set import *
+from kwiver.vital.types.activity import *
```

Comment:
As above, could a comment be add explaining that these lines are intentionally out of order and why that was done? Perhaps it could reference the comment it `CMakeLists.txt`.

Reply:
How does d73ddf01c8202f9a56bfb5c8e27ef9590b9daa55 look?

Reply:
It looks good to me, thanks! The only possible concern I have is that just saying `CMakeLists.txt` might be unclear or ambiguous to someone (is it the root one or the one in this folder?), but that's a minor issue at most.

Reply:
I agree. How about d6d1b5390864b4983602690c43baf549effa5ef6?

Reply:
Looks good to me.